api-reference/50 Common/Object Structures/ExportDataGridProps/customizeCell.md

```diff
@@ -24,3 +24,169 @@ An ExcelJS object that describes an Excel cell. Use its properties to customize
 A **DataGrid** cell.
 
 ---
+
+A snippet where Excel cells are customized 
+
+---
+##### jQuery
+
+    <!--JavaScript-->
+    $('#gridContainer').dxDataGrid({
+        export: {
+            enabled: true
+        },
+        onExporting(e) {
+            // ...
```

Comment:
You should show the file actually being saved because it would be problematic for the reader to find this information.